.maestro/tests/account/changeServer.yaml

```diff
@@ -1,5 +1,6 @@
 appId: ${APP_ID}
```

Comment:
![image](https://user-images.githubusercontent.com/1342360/222959054-5af9269b-9e07-40f7-abdb-bd828fa9b38f.png)

Note that while in dark mode, the page indicator icons (above the "Get started" button) are still not visible.

Reply:
![image](https://user-images.githubusercontent.com/1342360/222960305-59dbc146-b4c4-4c94-b723-28073f5dfdeb.png)
The background of replies are also very difficult to differentiate from the background of messages. The contrast between them is too low.

Reply:
And the tap-and-hold message of messages is white text on a white background.

![image](https://user-images.githubusercontent.com/1342360/222960515-d31d41c7-f9b6-487e-80c5-70a19ff97e90.png)

Reply:
Thanks for reporting these issues. However, I think they are out of scope for this PR: 

* The onboarding slideshow will be replaced once we have the right designs, if I'm not mistaken.
* The other issues are related to the timeline instead. Could you raise these in a different issue?